openshift/machine-config-operator
# OCPBUGS-43552: maxUnavailable value is not honored when disabling OCL

**- What I did**
Reworked `getUnavailableMachines` to remove assumptions about how OCL worked when we were attached to MCPs and configmaps.

**- How to verify it**
   1. Enable OCL in the worker pool
   2. Remove the MOSC resource to disable OCL
   
2 nodes should not drain at the same time

**- Description for the changelog**
